### ISSUE TITLE
GTA SA Mix-Mod Spawn Ped & Weapon Issues

### ISSUE THREAD

**Issue by lcmiracle** [user-grinch/GrinchTrainer-III-VC-SA#23]
So I know that the trainer is by no means obligated to work with another mod. I'm just asking here if there's a way to solve it.
I noted that in the \GrinchTrainerSA\data folder can be found peds.toml & weapons.toml, which I tried to edit with notepad.
I tried to match the IDs in the toml file with the weapons.dat and ped.dat in the gta-mixed/data folder, but it just shows in the menu the weapons are "UNKNOWN"
Using the original toml files will result in the spawn menu display "Invalid Model ID" error message.
One thing is I notice the mod seems to auto-detect the weapon.dat file because new weapons from the mod are appended to the end of the weapons.toml file as custom weapons. Can it be some sort of conflict?
Can this be fixed in the toml files itself?

**Comment by user-grinch** [comment]
Send you files here.

**Comment by lcmiracle** [comment]
Hi thanks for responding! So I actually deleted the old weapons.toml file since it didn't work, and forgot to make a backup.
But I can trying something similar with a handful of weapons:
[weapons.toml.zip](https://github.com/user-attachments/files/18048398/weapons.toml.zip)
So in the above I replaced every ID with the mixed mod's E value in weapon.dat
In testing all the default weapon names are displayed as "UNKNOWN" and can't be spawned.
I also replaced the modified weapon.xoml with the original from this repo, and the name displayed are correct but still can't spawn:
https://ibb.co/GR6v3h7
I'll also include the weapon.dat file and the weapon.dat file from the mixed-mod as well as the ped.ide (it does not have a ped.dat file strangely)
[gtamixeddata.zip](https://github.com/user-attachments/files/18048524/gtamixeddata.zip)
